### ISSUE TITLE
Addon not loading - Missing includes in embeds.xml

### ISSUE THREAD

**Issue by DoyleD** [SadRobotGG/BronCena#4]
Addon was failing to load and throwing Lua errors due to libs not being found. The AceAddon-3.0 and AceDB-3.0 libs were missing in the embeds.xml file.

Addin the below fixes the issue
```xml
<Include file="Libs\AceAddon-3.0\AceAddon-3.0.xml" />
<Include file="Libs\AceDB-3.0\AceDB-3.0.xml" />
```

Although i haven't done so myself it may be worth disabling all other addons except this one and checking if any other libs need referencing as well

**Comment by DavidMoore** [comment]
Thanks! That's a really good suggestion.

**Comment by DavidMoore** [comment]
Fixed in v1.4.3
